streaming/java/pom.xml

```diff
@@ -7,7 +7,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.ray</groupId>
   <artifactId>ray-streaming</artifactId>
-  <version>2.0.0-SNAPSHOT</version>
+  <version>0.0.1</version>
```

Comment:
Actually we shouldn't bump an official version to our developing branch. This patch need to be picked to a releasing branch and then we tag it.

Reply:
I knew it. But we want to merge extra changes except for version bump.

Reply:
OK, I see.  Approved. Don't forget to reversion to a snapshot as soon as possible on master branch then.